src/hotspot/os/linux/os_linux.cpp

```diff
@@ -2433,6 +2433,7 @@ void os::Linux::print_uptime_info(outputStream* st) {
   if (ret == 0) {
     os::print_dhm(st, "OS uptime:", (long) sinfo.uptime);
   }
+  assert(ret == 0, "sysinfo must return 0");
```

Comment:
Sorry we should report the value of `errno` if the assert fails
```suggestion
  assert(ret == 0, "sysinfo failed: %s", os::strerror(errno));
```